.gitignore

```diff
@@ -32,3 +29,9 @@ out/
 *.exe
 *.lib
 *.vcxproj.*
+subprojects/*
```

Comment:
Can you provide a brief description, in the commit-message, of why this is needed?

Reply:
Done